release/pkg/manager/operator/manager.go

```diff
@@ -98,16 +102,16 @@ func NewManager(opts ...Option) *OperatorManager {
 	return o
 }
 
-func (o *OperatorManager) Build(outputDir string) error {
+func (o *OperatorManager) Build() error {
 	if !o.isHashRelease {
 		return fmt.Errorf("operator manager builds only for hash releases")
 	}
 	if o.validate {
-		if err := o.PreBuildValidation(outputDir); err != nil {
+		if err := o.PreBuildValidation(o.tmpDir); err != nil {
```

Comment:
`o.PreBuildValidation` already has access to fields of `o`, no need to pass as an argument here.